funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -406,6 +515,23 @@ def stop_endpoint(*, name: str, remote: bool):
     _do_stop_endpoint(name=name, remote=remote)
 
 
+@app.command("update_funcx_config")
+@name_arg
+@click.option(
+    "--force",
+    is_flag=True,
+    default=False,
+    help="update config and backup to config.py.bak even if it already exists",
+)
+def update_funcx_endpoint_config(*, name: str, force: bool):
+    """
+    Update imports file from funcx_endpoint.* to globus_compute_endpoint.*
+
+    Either should raise ClickException or returns modification result message
+    """
+    print(_upgrade_funcx_imports_in_config(name=name, force=force))
```

Comment:
It's not a stopper, but I observe that this is the text that `click` uses for the command line `--help` parameter:
```
$ globus-compute-endpoint update_funcx_config --help
Usage: globus-compute-endpoint update_funcx_config 
           [OPTIONS] [NAME]

  Update imports file from funcx_endpoint.* to
  globus_compute_endpoint.*

  Either should raise ClickException or returns
  modification result message

Options:
  --force  update config and backup to config.py.bak even
           if it already exists
  --help   Show this message and exit.
```

Reply:
Will investigate to see if this can be avoided and --help shows appropriate message.